utils/_remote_config.py

```diff
@@ -36,41 +36,42 @@ def send_command(raw_payload, *, wait_for_acknowledged_status: bool = True) -> d
 
     client_configs = raw_payload["client_configs"]
 
+    current_states = {}
     if len(client_configs) == 0:
```

Comment:
could you update the docstring, explaining what is the data structure returned ?

Reply:
Yes of course, I forgot, thanks for the input